aspnetcore/performance/ObjectPool/ObjectPoolSample8/Program.cs

```diff
@@ -0,0 +1,58 @@
+using Microsoft.Extensions.DependencyInjection.Extensions;
+using Microsoft.Extensions.ObjectPool;
+using System.Security.Cryptography;
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.TryAddSingleton<ObjectPoolProvider, DefaultObjectPoolProvider>();
+
+builder.Services.TryAddSingleton<ObjectPool<ReusableBuffer>>(serviceProvider =>
+{
+    var provider = serviceProvider.GetRequiredService<ObjectPoolProvider>();
+    var policy = new DefaultPooledObjectPolicy<ReusableBuffer>();
+    return provider.Create(policy);
+});
+
+var app = builder.Build();
+
+app.MapGet("/", () => "Hello World!");
+
+// return the SHA256 hash of a word 
+// https://localhost:7214/SamsonAmaugo
+app.MapGet("/{name}", async (string name, ObjectPool<ReusableBuffer> bufferPool) =>
+{
+
+    var buffer = bufferPool.Get();
+    try
+    {
+        // Set the buffer data to the ASCII values of a word
+        for (int i = 0; i < name.Length; i++)
```

Comment:
```suggestion
        for (var i = 0; i < name.Length; i++)
```

Reply:
Hi @JamesNK. just for clarification purposes and learning. Please what was your reason behind switching to var